js/renovation/ui/scroll_view/scrollable_native.tsx

```diff
@@ -218,11 +220,12 @@ export class ScrollableNative extends JSXComponent<ScrollableInternalPropsType>(
   }
 
   get cssClasses(): string {
-    const { direction } = this.props;
+    const { direction, classes } = this.props;
 
     const classesMap = {
-      'dx-scrollview dx-scrollable dx-scrollable-native dx-scrollable-renovated dx-scrollable-native-generic': true,
+      'dx-scrollable dx-scrollable-native dx-scrollable-renovated dx-scrollable-native-generic': true,
       [`dx-scrollable-${direction}`]: true,
+      [`${classes}`]: !!classes,
```

Comment:
!! operator is used for null/undefined but I am sure that we don't want to add null/undefined to classes by the [`${classes}`] key

Reply:
I copyed this solution from existing code base. It uses widely. Look at the [widget.tsx file](https://github.com/DevExpress/DevExtreme/blob/7d29e0213722526f4d7e4be158778dfda68438e8/js/renovation/ui/common/widget.tsx#L49)

Reply:
ok, we use combineClasses that filters members by null/undefined values